extensions/ql-vscode/src/interface-utils.ts

```diff
@@ -137,7 +143,7 @@ export function getHtmlForWebview(
   <head>
     <meta http-equiv="Content-Security-Policy"
           content="default-src 'none'; script-src 'nonce-${nonce}'; style-src 'nonce-${nonce}'; connect-src ${webview.cspSource};">
-    <link nonce="${nonce}" rel="stylesheet" href="${stylesheetWebviewUri}">
+        ${stylesheetsHtmlLines.join(' ')}
```

Comment:
nit: Would this make the generated html easier to read?

```suggestion
        ${stylesheetsHtmlLines.join('\n    ')}
```

Reply:
I've tidied that up but used os.EOL instead of `\n`